lib/hypervisors.py

```diff
@@ -766,7 +783,7 @@ def test_stage(self, builder: Builder):
         cmd = "$Env:SMB_USERNAME = '{0}'; $Env:SMB_PASSWORD='{1}'; " \
```

Comment:
We need to inject for the Windows platform also:
`$Env:OS_MAJOR_VER`

Reply:
Resolved